client/jetpack-connect/happychat-button.jsx

```diff
@@ -22,6 +23,14 @@ const JetpackConnectHappychatButton = ( {
 	isChatAvailable,
 	translate,
 } ) => {
+	if ( ! isEnabled( 'jetpack/happychat' ) ) {
+		return (
+			<div>
+				{ children }
+			</div>
```

Comment:
Hmm, I'm missing a bit of context. By the PR desc, I'd expect the component to `return null`. What kind of `children` can this component have? Can we at least drop the `<div />`?

Reply:
Well, the component can have multiple items as `children`, which is why we have the `<div />` there. And it doesn't return null, because the expected behavior of this component is to return the `children` only if Happychat is not available (which is the case if the flag is turned off).

You can see #18397 for example `children`, and the button PR (#18396) for some more context.

Reply:
😩  Sigh. React<16 limitations.

We should be able to render arrays directly soon 🙏